logic-and-algorithm-test/3.py

```diff
@@ -1,25 +1,25 @@
 def generate_sequences(n_terms):
     # Sequence 1: Squares of natural numbers
-    seq1 = [i**2 for i in range(1, n_terms + 1)]
+    sequence1 = [i**2 for i in range(1, n_terms + 1)]
     
     # Sequence 2: n^2 - 2n + 2
-    seq2 = []
+    sequence2 = []
     for i in range (1, n_terms + 1):
```

Comment:
There is a typo in the method 'append'. It is written as 'apd' which will cause a runtime error. It should be corrected to 'append'.